libass/ass_filesystem.c

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright (C) 2021 Vabishchevich Nikolay <vabnick@gmail.com>
```

Comment:
Is it legally correct to change scheme only for individual files? Is that line enough and concrete definition of the term "libass contributors" (in the form of a list, for example) isn't needed? Personally I don't care about copyrights (I consider any form of information ownership a violation of fundamental right of sentient beings to process information) so any variant is fine for me.

Reply:
To my understanding, the `Copyright (C) ...` lines are not required by law, copyright is automatically obtained with the creation and so all contributors. So they are more informational, **however** our licence refers to and insist on the non-removal of those copyright lines, meaning we need to have *some* such line for the licence to make sense and we can't remove existing lines, only add new ones.
Therefore, it seems to me like the lack of a concise definition of “libass contributors” isn't problematic. In fact our `COPYING` file has already been using “libass contributors” ever since a copyright line was added to it in https://github.com/libass/libass/commit/9422a80578bc44f662f07edf723f85239a0579f5 .

Though, to make sure this actually helps and doesn't make things only more confusing, with individual and “libass contributors” lines mixed, we'd need to all agree on what to use in the future and what to do with existing files. *(E.g. use “libass contributors” for all new files, add “libass contributors” to existing files on edit and only bump the dates of those lines in the future.)*

Reply:
Replaced with `libass contributors`.